admin/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Admin project was being covered by Hubs prettierrc before. Fine to give it its own but we might not want to change the properties. since it will leed to A LOT of noise in the PR. If we really want to change it we should do just the formatting change in its own PR otherwise its pretty hard to look at the diff.